src/i18n/en/nav.ts

```diff
@@ -29,22 +29,22 @@ export default [
 	{ text: 'Static Assets', slug: 'guides/imports', key: 'guides/imports' },
 	{ text: 'Troubleshooting', slug: 'guides/troubleshooting', key: 'guides/troubleshooting' },
 	{ text: 'Deploy', slug: 'guides/deploy', key: 'guides/deploy' },
+	{ text: 'Server-side Rendering (SSR)', slug: 'guides/server-side-rendering', key: 'guides/server-side-rendering' },
 
 
 	{ text: 'Guides', header: true, type: 'learn', key: 'features' },
```

Comment:
The difference between the `Basics` section and the `Guides` section isn't that `Guides` are Guides. It's that `Guides` are "taking Astro beyond basics", incorporating other tools from _beyond_ Astro _into_ Astro: TypeScript, integrations, frameworks, styling solutions, fonts.

Now is a good opportunity to rename Guides to reflect that. Something as simple as this would be NWTWWHB:
```suggestion
	{ text: 'Next Steps', header: true, type: 'learn', key: 'features' },
```
```suggestion
	{ text: 'More', header: true, type: 'learn', key: 'features' },
```
It could have some flair:
```suggestion
	{ text: 'Beyond Basics', header: true, type: 'learn', key: 'features' },
```

Reply:
So, I'm not sure any of these are immediate no brainers, though I get the idea. And, I know these are just illustrative, but my criterion to you was "Do you have a no-brainer winning suggestion so we don't get bogged down with this" and I'm not sure we're quite at that level yet.  😄  If I put on Critical Cap:

"Next Steps" seems weird if I come into docs searching and I'm "already there." So, if I think about a variety of readers, not ones progressing in order, this doesn't resonate with me. I get what it means, but I think it could be worse than "Fake Guides."

"More" may be more accurate than Guides-that-aren't-really-Guides, but it totally feels like phoning it in. Looks lazy and sloppy.

"Beyond Basics" also feels a bit lazy. Basics! Beyond Basics! Even Less Basic?! I also think this hits a little worse.

I think "Guides" is a section heading that the average person is expecting. I also think the average person won't really notice that many of them aren't technically "guides." (Some of them sort of are!) I think people will see Core Concepts and get that those are a bit more conceptial. I think people will see Basics and think, "This is what I gotta know to wrap my head around Astro." And, I think people will see Guides and think, "These are natural extensions of the basic, core concepts of Astro. These are things I might want to figure out how to do." (e.g. add an RSS feed, set up TypeScript) So, I think Guides works? For now? 

I think coming up with something that tops these built-in advantages of the "strategic lie" of "Guides" is not as simple a task as it may seem. So, I think we should hold off on this for this PR to prioritize highlighting UI Frameworks, and maybe a couple of other things, more urgently.

Reply:
Right, I couldn't come up with a no-brainer suggestion. I just don't like Guides when both sections are equal amounts Guides.

Yes, let's hold it off. I _had_ to bring it up, and I'm glad you wrote out this analysis.